src/defaults/html.pipe.js

```diff
@@ -29,7 +30,7 @@ const validate = require('../utils/validate');
 
 /* eslint no-param-reassign: off */
 
-const htmlpipe = (cont, payload, action) => {
+const htmlpipe = async (cont, payload, action) => {
```

Comment:
Is this a required change ? side effects... ?

Reply:
it was always returning a promise. but using `async` ensures that it really is one. but let me check again. I think I only did it for the tests.